<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSettingsViewsPanel</class>
 <widget class="ctkSettingsPanel" name="qSlicerSettingsViewsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewsSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>449</width>
        <height>573</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
         <property name="title">
          <string>Slice viewer defaults</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="SliceOrientationMarkerLabel">
            <property name="text">
             <string>Orientation marker:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="ctkComboBox" name="SliceOrientationMarkerTypeComboBox">
              <item>
               <property name="text">
                <string>none</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cube</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>human</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>axes</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="ctkComboBox" name="SliceOrientationMarkerSizeComboBox">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>small</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>medium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>large</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="SliceRulerLabel">
            <property name="text">
             <string>Ruler:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ctkComboBox" name="SliceRulerTypeComboBox">
            <item>
             <property name="text">
              <string>none</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>thin</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>thick</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="SliceViewOrientationLabel">
            <property name="text">
             <string>View orientation:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ctkComboBox" name="SliceViewOrientationComboBox">
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_2">
         <property name="title">
          <string>3D viewer defaults</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="ThreeDBoxVisibilityLabel">
            <property name="text">
             <string>Show 3D cube:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ctkCheckBox" name="ThreeDBoxVisibilityCheckBox">
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ThreeDAxisLabelsVisibilityLabel">
            <property name="text">
             <string>Show 3D axis label:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ctkCheckBox" name="ThreeDAxisLabelsVisibilityCheckBox">
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="ThreeDOrientationMarkerLabel">
            <property name="text">
             <string>Orientation marker:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="ctkComboBox" name="ThreeDOrientationMarkerTypeComboBox">
              <item>
               <property name="text">
                <string>none</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cube</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>human</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>axes</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="ctkComboBox" name="ThreeDOrientationMarkerSizeComboBox">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>small</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>medium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>large</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="ThreeDRulerTypeLabel">
            <property name="text">
             <string>Ruler:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="ctkComboBox" name="ThreeDRulerTypeComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Ruler is only displayed if orthographic projection mode is used.</string>
            </property>
            <item>
             <property name="text">
              <string>none</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>thin</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>thick</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="ThreeDUseDepthPeelingLabel">
            <property name="text">
             <string>Use depth peeling:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="ctkCheckBox" name="ThreeDUseDepthPeelingCheckBox">
            <property name="toolTip">
             <string>Enabling depth peeling improves rendering of transparent models at the cost of higher computational cost.</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Use orthographic projection:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ctkCheckBox" name="ThreeDUseOrthographicProjectionCheckBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="MSAALabel">
           <property name="text">
            <string>Multi-sampling (MSAA):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="MSAAComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Use multisampling for full-screen anti-aliasing.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSettingsPanel</class>
   <extends>QWidget</extends>
   <header>ctkSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ThreeDUseOrthographicProjectionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ThreeDRulerTypeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
